js/src/elements/form-behavior.ts

```diff
@@ -27,6 +28,7 @@ export class FormBehaviorElement extends HTMLElement {
 	validators: Validator[] = [
 		this.validateMinLength.bind(this), // min length before required
 		this.validateRequired.bind(this),
+		this.validateGroupRequired.bind(this), // Checkbox or Radio groups
```

Comment:
Is this in the correct order? For radios, I do not see this running. It seems that for required radios, the validator on line 30 fails before we reach this.

Reply:
As you can see this results in different labels between select and radio.

<img width="376" alt="image" src="https://github.com/user-attachments/assets/4cf8e18d-b59a-4ed1-a2d9-2c1a16c37f2b">